### ISSUE TITLE
Warning: filemtime(): stat failed for ... /plugins/embed-privacy-dev/inc/class-fields.php

### ISSUE THREAD

**Issue by oldrup** [epiphyt/embed-privacy#209]
### Bug/Problem

Hey! 

Testing the current dev version, I get these two warnings when editing an embed:

```
Warning: filemtime(): stat failed for /example.net/wp-content/plugins/embed-privacy-dev/assets/js/admin/image-upload.min.js in /home/sustain2/oldrup.dk/wp-content/plugins/embed-privacy-dev/inc/class-fields.php on line 66

 Warning: filemtime(): stat failed for /example.net/wp-content/plugins/embed-privacy-dev/assets/style/embed-privacy-admin.min.css in /home/sustain2/oldrup.dk/wp-content/plugins/embed-privacy-dev/inc/class-fields.php on line 71
```

Screenshot:

![image](https://github.com/epiphyt/embed-privacy/assets/17141205/97634e0f-d1b5-4042-8d5b-14268ed7e57c)

Just double-checked - this warning does not occur with the current stable version from WordPress.org

The warnings do not interfere with the functionality it seems; I've tested two embed types in two languages on https://oldrup.dk/en/more-speed-less-tracking-with-embed-privacy/ - works well.

### Steps to reproduce

1. install 1.8.1 dev branch
2. edit an embed
3. might want to enable WP_DEBUG, not sure

### Version

1.8.1 dev branch

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MatzeKitt** [comment]
You’ve probably missed to build the minified JavaScript/CSS files.

To do so, run the following commands:

```
npm install
npm run build
```

**Comment by oldrup** [comment]
> You’ve probably missed to build the minified JavaScript/CSS files.

Yeah, I definitely missed to build anything because I just downloaded the dev zip and uploaded it to my site 😁 Honestly, I had no idea I needed to build anything, sorry about that.

Still, it works fine on the front end, with the 4 types of embeds I tested, on 3 different sites (same host though).

Please let me know if I can help in any other way, and thank you for all your hard work. It's very much appreciated 😍

**Comment by MatzeKitt** [comment]
Yeah, the readme here on GitHub definitely needs to be adjusted accordingly! Thank you for the heads up. 🙂
